test/cmdlineTests/standard_import_evmasm_immutable_references/output.json

```diff
@@ -0,0 +1,86 @@
+{
+    "contracts":
+    {
+        "A":
+        {
+            "":
+            {
+                "evm":
+                {
+                    "assembly": "    /* \"test.sol\":59:159   */
+  assignImmutable(\"0xe455bf8ea6e7463a1046a0b52804526e119b4bf5136279614e0b1e8e296a4e2d\")
+stop
+
+sub_0: assembly {
+        /* \"test.sol\":153:154   */
+      immutable(\"0xe455bf8ea6e7463a1046a0b52804526e119b4bf5136279614e0b1e8e296a4e2d\")
+
+    auxdata: <AUXDATA REMOVED>
+}
+",
+                    "bytecode":
+                    {
+                        "functionDebugData": {},
+                        "linkReferences": {},
+                        "object": "<BYTECODE REMOVED>",
+                        "opcodes":"<OPCODES REMOVED>",
+                        "sourceMap":"<SOURCEMAP REMOVED>"
+                    },
+                    "deployedBytecode":
+                    {
+                        "functionDebugData": {},
+                        "immutableReferences":
+                        {
+                            "6":
```

Comment:
So, is this correct? `6` is the value, but wasn't this supposed to be the variable name instead?

If this does not work as intended, then we're better off disabling this output.

Reply:
In the input we set that to `6`.. however, this should be correct.. I did not change that original import code

Reply:
Hmm... maybe you're right. I just tried to compile this:
```json
{
    "language": "Solidity",
    "sources": {
        "contract.sol": {
            "content": "contract C { uint immutable abc = 123; function f() public returns (uint) { return abc;} }"
        }
    },
    "settings": {"outputSelection": {"*": { "*": ["evm.bytecode", "evm.deployedBytecode", "evm.assembly"]}}}
}
```

I got this in `immutableReferences`:
```json
                        "immutableReferences":
                        {
                            "3":
                            [
                                {
                                    "length": 32,
                                    "start": 71
                                }
                            ]
                        },
```
and this in `legacyAssembly`:
```json
                                    {
                                        "begin": 83,
                                        "end": 86,
                                        "name": "PUSHIMMUTABLE",
                                        "source": 0,
                                        "value": "3"
                                    },
```

So looks like it's not the name like I assumed. It refers to something else and it's apparently stored as `value` in assembly.